samza-log4j2/src/main/java/org/apache/samza/logging/log4j2/StreamAppenderMetrics.java

```diff
@@ -46,7 +43,6 @@ public class StreamAppenderMetrics extends MetricsBase {
   public StreamAppenderMetrics(String prefix, MetricsRegistry registry) {
     super(prefix + "-", registry);
     bufferFillPct = newGauge("buffer-fill-percent", 0);
-    recursiveCalls = newCounter("recursive-calls");
```

Comment:
is this metric published in some Samza doc? If yes, let's remove it from there as well.

Reply:
Didn't find StreamAppenderMetrics published in any oss samza doc
https://samza.apache.org/learn/documentation/latest/container/metrics-table.html